src/Commands/Backup/Schedule/SetCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Backup\Schedule;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+/**
+ * Class SetCommand
+ * @package Pantheon\Terminus\Commands\Backup\Schedule
+ */
+class SetCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Sets up a week-long TTL backups to be made daily and a month-long TTL to be made weekly
+     *
+     * @authorized
+     *
+     * @command backup:schedule:set
+     *
+     * @param string $site_env Site & environment to set the schedule of, in the format `site-name.env`.
+     * @option string  $day  Day of the week to make the month-long backup
```

Comment:
Can you give example here too guess it's a string so like `mon` vs `monday` vs `Monday`